src/midcom/datamanager/template/form.php

```diff
@@ -81,7 +81,7 @@ public function form_row(FormView $view, array $data)
         $string .= $this->renderer->label($view);
         $string .= $this->renderer->errors($view);
         $string .= '<div class="input">';
-        $string .= $this->renderer->widget($view);
+        $string .= $this->renderer->humanize($this->renderer->widget($view));
```

Comment:
this would try to translate a rendered widget, i.e. an HTML string. Is this really necessary for something?